examples/specs/stacked_bar_opacity.json

```diff
@@ -0,0 +1,24 @@
+{
+  "data": {"url": "data/seattle-weather.csv", "formatType": "csv"},
+  "mark": "bar",
+  "encoding": {
+    "x": {
+      "field": "date",
+      "type": "temporal",
+      "timeUnit": "month"
+    },
+    "y": {
+      "field": "*",
+      "type": "quantitative",
+      "aggregate": "count"
+    },
+    "color": {
+      "field": "weather",
+      "type": "nominal"
+    },
+    "opacity": {
```

Comment:
Also, encoding nominal field with opacity channel is a bad example, so I think we should change example here.